concepts/numbers/about.md

```diff
@@ -1,8 +1,50 @@
 # About
 
-## Type Conversion
+Go contains basic numeric types that can represent sets of either integer or
+floating-point values. There a different types depending on the size of value
+you require and the architecture of the computer where the application is
+running (e.g. 32-bit and 64-bit).
+
+For the sake of this exercise you will only be dealing with:
+
+- `int`: e.g. `0`, `255`, `2147483647`. A signed integer that is at least 32
+  bits in size (value range of: -2147483648 through 2147483647). But this will
+  depend on the systems architecture. Most modern computers are 64 bit,
+  therefore `int` will be 64 bits in size (value rate of:
+  -9223372036854775808 through 9223372036854775807).
+
```

Comment:
This seems to be missing an explanation of the other int types: https://tour.golang.org/basics/11

In particular, that you can get an explicit 64-bit version using int64 and uint64.

Reply:
I would argue not to mention the other integer types. The goal of this exercise is very simple: teach students how to use the two most-used numeric types: `int` and `float64`. There is so much more that can be taught about integers and floats that there should be separate exercises to teach that. For example, C# has the [hyper-optimized-telemetry exercise](https://github.com/exercism/csharp/tree/main/exercises/concept/hyper-optimized-telemetry) to teach about all the integer types, and the [interest-is-interesting exercise](https://github.com/exercism/csharp/tree/main/exercises/concept/interest-is-interesting) to teach about all the floating point types.